imhotep/reporters/reporters_test.py

```diff
@@ -24,6 +29,30 @@ def test_pr_url():
     )
 
 
+def test_pr_review_reporter_should_add_comments():
+    requester = Requester("")
+    pr = PRReviewReporter(requester, "github.com", "justinabrahms/imhotep", 10)
+    pr.report_line(commit="sha", file_name="script.py", position=0, message="lorem")
+    assert pr.comments == [{"body": "* lorem\n", "path": "script.py", "position": 0}]
+    pr.report_line(commit="sha", file_name="script.py", position=1, message="ipsum")
+    assert pr.comments == [
+        {"body": "* lorem\n", "path": "script.py", "position": 0},
+        {"body": "* ipsum\n", "path": "script.py", "position": 1},
+    ]
```

Comment:
Can you also assert that the "publish review" thing wasn't called internally?

Reply:
Done.